russian/reglamentation-zefir/reglamentation-zefir-againstCrime/reglamentation-zefir-againstCrime.md

```diff
@@ -0,0 +1 @@
+Работая на международную полицию, вы узнаете про множество недобросовестных дел, которые делает правительство стран мира. Ваше чувство справедливости обретает новые силы и вы строите план, чтобы скрыть все тайны. Вскорее, вам будут благодарны люди многих стран мира за то, что вы облагоразумили их на смену власти и станете международным героем ...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.